src/feature/stats/bw_array.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2007-2020, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+/**
+ * @file bw_array_st.h
+ * @brief Bandwidth array manipulation functions.
+ **/
+
+#ifndef BW_ARRAY_H
+#define BW_ARRAY_H
+
+#include "feature/stats/bw_array_st.h"
```

Comment:
Consider replacing the struct declaration with a forward declaration:
```suggestion
struct bw_array_t;
```

Reply:
Suggestion implemented.